index.js

```diff
@@ -8,6 +8,9 @@ module.exports = {
 
   parserOptions: {
     ecmaVersion: 2018,
+    ecmaFeatures: {
+      experimentalObjectRestSpread: true,
```

Comment:
Looking at this now in context: are you sure you're running the right version of eslint? According to https://eslint.org/docs/user-guide/configuring#deprecated, "This syntax has been supported in ecmaVersion: 2018. This option will be removed in the future." So with `ecmaVersion: 2018` on L10 above this should be unnecessary (and indeed we've been using this feature in a lot of places without linter complaints so far). @jalack I recall having had this conversation with you before, do you remember what the issue was?

Reply:
I should've known something was a bit off, updating `eslint` to `4.19.1` (or at least newer than `4.18.1`) fixes this . I will close this PR.